src/picturepark-sdk-v1-angular/projects/picturepark-sdk-v1-angular-ui/src/lib/features-module/list-browser/list-browser.component.ts

```diff
@@ -124,7 +124,14 @@ export class ListBrowserComponent extends BaseBrowserComponent<ListItem> impleme
       limit: this.pageSize,
       searchString: this.searchString,
       sort: this.sortInfo,
-      searchBehaviors: [SearchBehavior.DropInvalidCharactersOnFailure, SearchBehavior.WildcardOnSingleTerm],
+      searchBehaviors: this.searchBehavior ? [
```

Comment:
In my opinion, this would look cleaner if it was declared in a variable before this request creation. 
For example something like this:

```
const searchBehaviors = this.searchBehavior ? [this.searchBehavior] : [];
searchBehaviors.push(SearchBehavior.DropInvalidCharactersOnFailure,
    SearchBehavior.WildcardOnSingleTerm);
```